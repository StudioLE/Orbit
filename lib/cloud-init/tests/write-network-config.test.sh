#!/bin/bash
set -uo pipefail

# IMPORTS
source /srv/lib/cloud-init/write-network-config.sh

# ARGS
ACTUAL_FILE=$1

# START
OUTPUT=""
write-network-config 253
echo "${OUTPUT}" > "${ACTUAL_FILE}"
echo "${OUTPUT}"
