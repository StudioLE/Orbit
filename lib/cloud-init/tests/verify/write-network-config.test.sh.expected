#cloud-config

network:
  version: 2
  ethernets:
    eth0:
      set-name: eth0
      match:     
        name: en*
      addresses:
        - 10.10.10.253/24
      nameservers:
        addresses:
          - 10.10.10.1
          - 1.1.1.1
          - 1.0.0.1
        search:
          - local
      routes:
        - to: default
          via: 10.10.10.1
